spec/index.html

```diff
@@ -619,7 +619,7 @@ <h3>Grammar</h3>
           match the pattern for <a href="#grammar-production-LANGTAG">LANGTAG</a>,
           though neither "<code class="grammar-literal">prefix</code>"
           nor "<code class="grammar-literal">base</code>"
-          are <a data-cite="?LANG-SUBTAG-REGISTRY#">registered language subtags</a>.
+          are <a data-cite="?LANG-SUBTAG-REGISTRY#">registered language subtags</a> [[LANG-SUBTAG-REGISTRY]].
```

Comment:
Should the data-cite also be normative?
```suggestion
          are <a data-cite="LANG-SUBTAG-REGISTRY#">registered language subtags</a> [[LANG-SUBTAG-REGISTRY]].
```